### PR TITLE
Update USA Banner component

### LINKED ISSUE #33: UI: USA Banner + Dropdown template
Create a `templ` template for the USA Banner ("An official government website...") to match the CDC styles (based on Heather's [mockup](https://www.figma.com/design/2qu1dogZOKe4qXW1QEhEJh/PHIN-VADS-Prototype?node-id=41-1585&t=CejJahZOAqO0hPxg-0) and the CDC [home page](https://www.cdc.gov/))
- split out existing code from the `Base` template and put it in its own file

### PR DESCRIPTION
**Pull request by katyasoup**
resolves #33 

## What this PR changes:
- adds a new route to enable `htmx` swapping for show/hide functionality on the USA banner dropdown content
  - updates templ template to accept a parameter for showing/hiding the banner contents
- updates "Here's how you know" clickable element:
  - replaces the `<a>` element with a `<button>` element for better keyboard navigability
- updates styles to match the banner at [https://www.cdc.gov/phin/php/vocabulary/](https://www.cdc.gov/phin/php/vocabulary)

## How to test:
- Test at both full browser screen width (> 1000px) and smaller responsive widths ( < 900px)
- Click "Here's how you know" and make sure the content properly shows/hides and styles do not change
- Make sure you can interact with the clickable element with both a mouse/trackpad click, as well as tabbing to it and clicking with the space/enter key

## Notes:
- Keyboard navigability needs additional work; when using the keyboard to show/hide the banner content, focus should remain on the button element; currently you need to tab backwards to be able to focus and click it again. I'll add another [ticket](https://github.com/skylight-hq/phinvads-go/issues/39) for this.

## Feedback requested:
- ~How can we use the `Templ` templating instead of creating additional HTML files to use with HTMX functionality?~ addressed in [99cbe18](https://github.com/skylight-hq/phinvads-go/pull/37/commits/99cbe1879d5fb926c8051d0c724957fa458e84f3)

### CODE REVIEW

**Comment by katyasoup** [comment]
@nickclyde updated [d2aac37](https://github.com/skylight-hq/phinvads-go/pull/37/commits/d2aac37b88e15cce3911256735599599764f5def) per our convo!

File: internal/app/handlers.go [hunk 1]
```diff
@@ -425,9 +425,9 @@ func (app *Application) handleBannerToggle(w http.ResponseWriter, r *http.Reques
 	action := r.PathValue("action")
 	var component templ.Component
 	if action == "close" {
-		component = components.UsaBanner()
+		component = components.UsaBanner("close")
```
Comment (nickclyde) [inline internal/app/handlers.go:428]: This looks great! You could simplify it even further by removing this if statement and just doing `component = components.UsaBanner(action)`!

Comment (katyasoup) [reply internal/app/handlers.go]: 💡 done!

**Review by nickclyde — APPROVED** [review]
Looks great!!

**Comment by akintner** [comment]
LGTM 🚀

**Pull request merged**